src/include/binder/binder.h

```diff
@@ -121,6 +121,8 @@ class Binder {
         const std::vector<parser::ParsedPropertyDefinition>& parsedDefinitions,
         const std::string& tableName);
 
+    static void validatePropertyName(const std::vector<PropertyDefinition>& definitions);
```

Comment:
Remove this. Let's try not add static function to header. Hide them in cpp file.